components/switch/switch-mixin.js

```diff
@@ -208,7 +215,7 @@ export const SwitchMixin = superclass => class extends FocusMixin(RtlMixin(Focus
 	}
 
 	get _labelContent() {
-		return html`${this.text}`;
+		return html`<span @click='${this._handleClick}' @mouseover='${this._handleSwitchHover}' @mouseout='${this._handleSwitchHoverLeave}'>${this.text}</span>`;
```

Comment:
Generally, I think [mouseenter](https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseenter_event) and [mouseleave](https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseleave_event) would be better events to use for this purpose... combined, they act more similar to `:hover` in CSS than when using `mouseover` and `mouseout`.  There can be performance implications with deep hierarchies as well, as described in the Mozilla docs.

In this particular case it doesn't make any difference (in terms of the number of times the handlers are called) since the element only contains plain text, but it would if someone happened to add nested span to the label. It would still be minor, but in other contexts it can become more meaningful.